nipoppy/tabular/bagel.py

```diff
@@ -115,8 +110,9 @@ class Bagel(BaseTabular):
 
     _metadata = BaseTabular._metadata + [
         "col_participant_id",
-        "col_bids_id",
+        "col_bids_participant",
         "col_session_id",
+        "col_bids_session",
```

Comment:
out of interest: why not `col_bids_participant_id`? That would be consistent with `col_participant_id`

Reply:
I think we wanted to use the `_id` suffix for what BIDS sometimes refers to as the label. Also `col_bids_participant_id` is somewhat long.

At one point we had `participant_id` and `bids_id` which kind of worked but there was no equivalent/parallel way to name the session IDs.

I am not really opposed to renaming `bids_participant` -> `bids_participant_id` and `bids_session` -> `bids_session_id`, although that will again be a breaking change so we should decide soon if we want to do this.

Reply:
I'd vote for consistency/clarity over brevity. The machines don't care, but from time to time a human might look at it and they would appreciate it.